packages/create-discord-bot/template/index.js

```diff
@@ -0,0 +1,9 @@
+import { Client, Events, GatewayIntentBits } from 'discord.js';
+
+const client = new Client({ intents: [GatewayIntentBits.Guilds] });
+
+client.on(Events.ClientReady, () => {
+	console.log(`Logged in as ${client.user.tag}.`);
+});
```

Comment:
This is still in draft so I'm not sure will u add commands or not but adding 1 command will be helpful

Reply:
Iterations can be done in the future. Adding a command handler is on the list, so this is a needed addition!